README.md

```diff
@@ -43,6 +43,8 @@ branch](https://github.com/leafo/moonscript/tree/binaries)
 * [Textadept](https://github.com/leafo/moonscript-textadept)
 * [Sublime/Textmate](https://github.com/leafo/moonscript-tmbundle)
 * [Emacs](https://github.com/k2052/moonscript-mode)
+* [Visual Studio Code](https://github.com/Microsoft/vscode)
```

Comment:
can we link directly to the plugin instead of the editor: https://marketplace.visualstudio.com/items?itemName=Lynxish-Developer.moonscript